### PR TITLE
Create fields for calendar and messaging settings v2

### PR DESCRIPTION
**Pull request by bosiraphael**
Create fields for calendar and messaging settings v2

### CODE REVIEW

File: packages/twenty-server/src/modules/calendar/standard-objects/calendar-channel.workspace-entity.ts [hunk 1]
```diff
@@ -39,6 +39,13 @@ export enum CalendarChannelSyncStage {
   FAILED = 'FAILED',
 }
 
+export enum CalendarChannelContactAutoCreation {
```
Comment (charlesBochet) [inline packages/twenty-server/src/modules/calendar/standard-objects/calendar-channel.workspace-entity.ts:42]: I feel we are the naming could be improved
CalendarChannelContactAutoCreationPolicy or CalendarChannelContactAutoCreationMode

File: packages/twenty-server/src/modules/messaging/common/standard-objects/message-channel.workspace-entity.ts [hunk 2]
```diff
@@ -51,6 +51,12 @@ export enum MessageChannelType {
   SMS = 'sms',
 }
 
+export enum MessageChannelContactAutoCreation {
```
Comment (charlesBochet) [inline packages/twenty-server/src/modules/messaging/common/standard-objects/message-channel.workspace-entity.ts:54]: same here

**Review by charlesBochet — COMMENTED** [review]
Left comments about the naming!

File: packages/twenty-server/src/modules/calendar/standard-objects/calendar-channel.workspace-entity.ts (hunk 1 above)
Comment (bosiraphael) [reply packages/twenty-server/src/modules/calendar/standard-objects/calendar-channel.workspace-entity.ts:42]: I agree

**Review by Weiko — APPROVED** [review]

**Pull request merged**